### DISCUSSION TITLE
How to update nodes after a new release?

### DISCUSSION THREAD

**Discussion by artemdzyuba** [FlowiseAI/Flowise#694] [Q&A] [answered]
After latest update I see this:
![image](https://github.com/FlowiseAI/Flowise/assets/20184727/229d3634-45fe-4bb6-8ac0-f3b9fd515944)

How to update the nodes? I see no options.

**Comment by marius-smg** [comment, marked as answer]
I stumbled across this as well, as of now i couldnt find an option to just update, so it seems we have to do it manually.

**Reply by HenryHengZJ** [reply]
you'll have to redrag from the menu
